### PR TITLE
CE-3388  

### PR DESCRIPTION
**Pull request by crimsonred**
Ensure the setPresenceTimeout and setPresenceTimeoutWithCustomInterval values can not be lower than 20

### CODE REVIEW

File: examples/cli/cli_demo.go [hunk 1]
```diff
@@ -55,9 +55,9 @@ func connect() {
 	//config.Log = log.New(os.Stdout, "", log.Ldate|log.Ltime|log.Lshortfile)
 	config.Log = infoLogger
 	config.Log.SetPrefix("PubNub :->  ")
-	config.PublishKey = "demo"
-	config.SubscribeKey = "demo"
-	config.SecretKey = "demo"
+	config.PublishKey = "pub-c-c1648ded-d156-4a2d-9dbb-a23262945fe2"
```
Comment (davidnub) [inline examples/cli/cli_demo.go]: I assume we shouldn't be including these keysets in here?

File: tests/e2e/helper.go [hunk 2]
```diff
@@ -30,13 +30,14 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 
 	config = pubnub.NewConfig()
-	config.PublishKey = "pub-c-afeb2ec5-45e9-449f-9a8d-c4940a9c7836"
-	config.SubscribeKey = "sub-c-e41d50d4-43ce-11e8-a433-9e6b275e7b64"
+	config.PublishKey = "pub-c-c1648ded-d156-4a2d-9dbb-a23262945fe2"
```
Comment (davidnub) [inline tests/e2e/helper.go]: I wish this didn't have to be hardcoded. :(

File: examples/cli/cli_demo.go (hunk 1 above)
Comment (crimsonred) [reply examples/cli/cli_demo.go]: removed

File: tests/e2e/helper.go (hunk 2 above)
Comment (crimsonred) [reply tests/e2e/helper.go]: required for integration tests, the keys will be disabled after deploy.

Comment (crimsonred) [reply tests/e2e/helper.go]: Will make changes after the current tasks.

**Pull request merged**